Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM bitwalker/alpine-elixir-phoenix:1.9.4 as build
+COPY . /opt/app
+WORKDIR /opt/app
+RUN mix do local.hex --force, local.rebar --force, deps.get --only prod, deps.compile
+ENV MIX_ENV=prod
+RUN mix release orchestrate \
+  && mix release gather \
+  && mix release broadcast \
+  && mix release persist \
+  && mix release acquire
+
+FROM bitwalker/alpine-erlang:22.2.3
```

Comment:
I assume this is smaller but do we miss out on any debugging tooling by not running these in the `alpine-elixir` image?

Reply:
I don't think so. I think we'll have more available to us than the `smartcitiesdata` images have.

Reply:
👍